src/vm/stablehashcode.cpp

```diff
@@ -0,0 +1,137 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "common.h"
+#include "stablehashcode.h"
+#include "typehashingalgorithms.h"
+
+// DIFFERENT FROM CORERT: Hashing code currently different from CoreRT
```

Comment:
Could you please add this comment to the specific lines that differ? I believe it should be mostly the same ... there are just a few details that differ.